### PR TITLE
Fix dark theme dialog colors.

### LINKED ISSUE #167: When selecting a date for a new calendar event, the date dialog is not adjusted for dark mode
image

### LINKED ISSUE #168: When selecting a time for the homework reminders, the time dialog is not adjusted for dark mode
image

### LINKED ISSUE #169: When selecting a time for the timetable start, the time dialog is not adjusted for dark mode
image

Maybe related to #168

### LINKED ISSUE #384: The border of text fields it's not good visible in dark mode
| How it should be (current version from PlayStore) | How it currently is (current `main`) |
|-|-|
| telegram-cloud-photo-size-2-5463272927240768560-y | telegram-cloud-photo-size-2-5463272927240768559-y |

### LINKED ISSUE #496: "Erstellen" button for school classes has in dark mode the wrong color
image

### PR DESCRIPTION
**Pull request by Jonas-Sander**
By using `ColorScheme.fromSwatch().copyWith(...)` the `ColorScheme` was tailored to light-mode which caused some dialogs to be displayed in the wrong colors when using dark mode.
Specifying the `Brightness` inside `ColorScheme.fromSeed` fixed this issue.

|  Before  | After  |
|---|---|
| grafik  | grafik  |
| grafik  | grafik  | |grafik |  grafik |
|grafik  | grafik

Fixes #167 
Fixes #168
Fixes #169
Fixes #384 
Fixes #496

### CODE REVIEW

**Comment by nilsreichardt** [comment]
This PR also fixes #384

**Review by nilsreichardt — COMMENTED** [review]
Screenshot 2023-04-05 at 22 57 51

Snackbar in dark mode is not readable

**Comment by nilsreichardt** [comment]
image

Wrong in the PR description (you need to swap the images)

**Review by nilsreichardt — CHANGES_REQUESTED** [review]
LGTM

The only thing that I have seen was that the snackbar text has the wrong color in dark mode

File: app/lib/widgets/apple_sign_in_button.dart [hunk 1]
```diff
@@ -72,6 +72,7 @@ class SignInWithAppleButton extends StatelessWidget {
                   child: Icon(
                     FontAwesomeIcons.apple,
                     size: 26,
+                    color: Colors.white,
```
Comment (Jonas-Sander) [inline app/lib/widgets/apple_sign_in_button.dart:75]: As you can see in your last screen shot the icon was not completely white. I didn't find see this for the other icons in the app though.

**Review by nilsreichardt — APPROVED** [review]
LGTM

**Pull request merged**